cluster/ci/config/prow/config.yaml

```diff
@@ -1732,3 +1732,13 @@ periodics:
   labels:
       master: ci.openshift.redhat.com
   interval: 2h
+- name: azure_build_base_image
+  agent: jenkins
+  labels:
+      master: ci.openshift.redhat.com
+  interval: 24h
+- name: azure_build_node_image
+  agent: jenkins
+  labels:
+      master: ci.openshift.redhat.com
+  interval: 24h
```

Comment:
And this is questionable as well. We build node image for every pr test separately. This image will be used only for publishing, so.. @kwoodson what would be the proper interval here?

Reply:
if we're going to use it what's the harm? 

My main objective: getting **all** the process in place so we can learn and adjust them as necessary.  I'd rather be too aggressive and scale back rather than find we weren't doing something we should've been.

Knowing we can release nightly would be to our advantage.

Reply:
I think every 24 hours is overkill.  Important RHEL updates usually land once every 1-2 weeks.

Now having said that it obviously won't hurt us in terms of process.  The downside I imagine is storing those images.  We need to think about image retention/clean up.  I'd imagine we'd keep around 14 days (1-2 images if weekly and 14 if daily).

To summarize:
Pros:
- build process tested often
- RPM updates minimized (job should run very quickly if os updates are zero)
- Make @jim-minter @pweil-  happy
- always up-to-date
Cons:
- storage costs
- cost of process (pretty minimal)
- storage clutter
- always up-to-date  (could be less stable.  rhel7.5 came out recently and those . releases always wreak havoc on something)

I'm fine with a brisk building pace and a back off plan once things settle.  Maybe make a note of it.

Reply:
Updated to run both jobs weekly until we have GC in place.